README.md

```diff
@@ -35,3 +43,27 @@ This command generates static content into the `build` directory and can be serv
 ### Deployment
 
 Deployment is fully automatic and thus requires no configuration or user interaction.
+
+
+## Contributing to Yoast Developer portal
+Howdy! It's really great that you want to contribute to the Yoast developer portal. The development of the portal takes place on GitHub.
+
+There are three main ways to contribute:
+1. [Contributing to issues](#contributing-to-issues)
+2. [Reviewing pull requests](#reviewing-pull-requests)
+3. [Contributing to code](#contributing-to-code)
+
+### Contributing to issues
+We use Jira to track changes and improvements to the portal that need to be discussed and completed. The portal related tickets are [in this board](#).
+
+Before creating a ticket, please check if the issue you want to raise have not already received a ticket.
+
+Please be as descriptive and succinct as possible in your ticket, and if necessary, to help better understand the issue, add screenshots or video captures.
+
+### Reviewing pull requests
```

Comment:
Not sure if you wanted to add information here, having a header without copy seems weird :)